ui/src/components/SchemaForm/Widgets/RadioWidget.tsx

```
@@ -0,0 +1,91 @@
+
+  return (
+    <RadioGroup>
+      {Array.isArray(enumOptions) &&
+        enumOptions.map((option, index) => {
+          const itemDisabled =
+            Array.isArray(enumDisabled) &&
+            enumDisabled.indexOf(option.value) !== -1;
+          const checked = enumOptionsIsSelected<S>(option.value, value);
+
+          return (
+            <div className="flex items-center space-x-2">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)